Code cleanup

ArthurAtangana opened the pull request:
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [X] 📝 Documentation Update
- [ ] ✅ Test
- [x] 🧑‍💻 Refactor

## Description

i cant sleep so i cleanup up the codebase.
also fixed the imports and packages
merge after PR #100 

## How to test
 press run

## Added/updated tests?

- [ ] Yes
- [X] No, and this is why: its almost 3am
- [ ] I need help with writing tests:.

## Screenshots
![nosleep](https://github.com/ArthurAtangana/SYSC3303A_Project/assets/83664112/4467bb1c-9ef6-44df-ac84-21e6d08eb4cf)

mpdesantis reviewed (approved):
Very slick, Git God and GUI Guy.  Things run on my end!

ArthurAtangana committed:
Cleanup the codebase

- fix package imports for logger
- remove unused imports
- remove extra blank lines
- remove unused methods

ArthurAtangana committed:
Add closing the door timing and other config variables

ArthurAtangana committed:
Add opening the door and fix separate unloads for transient and regular passenger events.

ArthurAtangana committed:
re-add input file from config

ArthurAtangana committed:
Update test to remove 0 as a floor.

braedenkloke reviewed (approved):
No features included in this PR. Code looks great!

# Test
Tested scenario 6 (combined faults). System working as expected.
